tests/unit/test_commands.py

```diff
@@ -18,7 +18,16 @@
 import pytest
 
 from cou import commands
-from cou.commands import CLIargs
+from cou.commands import CONTROL_PLANE, DATA_PLANE, CLIargs
+
+
+@pytest.mark.parametrize(
+    "upgrade_group, expected_result",
+    [(CONTROL_PLANE, True), (DATA_PLANE, False), ("foo", True)],
```

Comment:
Hmm, "foo" is control plane? How we will use this?

Reply:
I changed the logic so we are not using it anymore